exodus_core/analysis/test_exodus_analyze.py

```diff
@@ -49,7 +49,7 @@ def test_trackers_list(self):
 
     def test_icon_diff(self):
         phash_4 = phash('./apks/nextcloud.apk')
-        self.assertEqual(phash_4, 325352301465779383961442563121869825536)
```

Comment:
Do we know why the phash has changed ? I'm guessing NextCloud just changed their app icon.

Reply:
Two goals:

  1-Yes the path to the icon has changed.

  2-Also that until now the function phash() used to take the hash of the icon in a way that changed the size to reduce it. It takes now the full size of the image.